src/scss/components/choice-tile/_index.scss

```diff
@@ -54,6 +56,24 @@ hx-tile {
     }
   }
 
+  &.hxSlim {
+    height: 64px;
+    width: 221px;
```

Comment:
Please use `rem` units.  Only go to X.XX rem for values.

Reply:
Changes have been done, Please review